migtests/tests/pg/partitions/validate

```diff
@@ -96,65 +63,102 @@ TABLE_NAME_TO_PARTITION_COUNT = {
 	"public.customers": 2
 }
 
-
+# Validation script will insert and check the incremented row count in partition table and to revert the change DELETE the row
 INSERT_QUERIES_INTO_TABLE_VERIFY_PARTITION = {
 	"List" : {
-		"query": "INSERT INTO sales_region  (id, amount, branch, region) VALUES (1001, 2000, 'Branch 1001','Sydney') ",
+		"insert_query": "INSERT INTO sales_region  (id, amount, branch, region) VALUES (1001, 2000, 'Branch 1001','Sydney') ",
+		"delete_query": "DELETE FROM sales_region WHERE id = 1001",
 		"partition_table_name": "sydney",
 		"schema_name": "public"
 	},
 	"List" : {
```

Comment:
Having "List" twice looks like a bug. The first row might be overwritten. We should rename it for the individual keys.
"List_sales_region", "List_test_partitions_sequences"

Reply:
This has been like this already.

can you tell me the exact replacements you are asking so that i dont end up breaking anything here